index.php

```diff
@@ -2067,10 +2116,20 @@ function getUrlParameters() {
 			Helioviewer.mobile = <?php echo isset($_SERVER['HTTP_USER_AGENT']) && (strpos($_SERVER['HTTP_USER_AGENT'],'Phone')|strpos($_SERVER['HTTP_USER_AGENT'],'Android')|strpos($_SERVER['HTTP_USER_AGENT'],'iPad')) ? 'true' : 'false'; ?>
 
 
+
 			const loadHelioviewer = (userSettings) => {
 
 				Helioviewer.userSettings = userSettings;
 
+                Helioviewer.eventLoader = FullEventLoader.make(Helioviewer.outputType, Helioviewer.debug)
```

Comment:
```suggestion
                Helioviewer.eventLoader = EventLoader.make(Helioviewer.outputType, Helioviewer.debug)
```
This is a little confusing because `make` is defined on EventLoader, but FullEventLoader extends EventLoader so this code technically works. But eventLoader can be either FullEventLoader or MinimalEventLoader. This code makes it look like it will always be FullEventLoader.